### PR TITLE
docs: updated the benchpress readme doc

### PR DESCRIPTION
**Pull request by alkavats1**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/benchpress/README.md [hunk 1]
```diff
@@ -236,13 +236,13 @@ To collect these metrics, you need the following corresponding extra providers:
   - metrics that are dependent on the performance of the execution environment must be executed on a normalized machine
   - e.g. a real mobile device whose cpu frequency is set to a fixed value.
       * see our [build script](https://github.com/angular/angular/blob/2.4.9/scripts/ci/android_cpu.sh)
-      * this requires root access, e.g. via a userdebug build of Android on a Google Nexus device
+      * this requires root access, e.g. via an userdebug build of Android on a Google Nexus device
```
Comment (jessicajaniuk) [inline packages/benchpress/README.md:239]: This is actually incorrect as the sound of the word following is pronounced like "you". It should be `a` here.

File: packages/benchpress/README.md [hunk 2]
```diff
@@ -236,13 +236,13 @@ To collect these metrics, you need the following corresponding extra providers:
...
         (see [here](https://source.android.com/source/building) and [here](https://source.android.com/source/building#obtaining-proprietary-binaries))
   - e.g. a calibrated machine that does not run background jobs, has a fixed cpu frequency, ...
 
 * Use relative comparisons
   - relative comparisons are less likely to change over time and help to interpret the results of benchmarks
-  - e.g. compare an example written using a ui framework against a hand coded example and track the ratio
+  - e.g. compare an example written using an ui framework against a hand coded example and track the ratio
```
Comment (jessicajaniuk) [inline packages/benchpress/README.md:245]: Same with this case.

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]
This is mostly ok, but the grammar is incorrect in two places.

File: packages/benchpress/README.md (hunk 2 above)
Comment (alkavats1) [reply packages/benchpress/README.md]: Hi @jessicajaniuk , resolved. Please review now.

**Review by jessicajaniuk — APPROVED** [review]
Thanks for the fix!

**Comment by jessicajaniuk** [comment]
@alkavats1 the broken tests should be fine after a rebase.

**Comment by alkavats1** [comment]
closing this will create new PR

**Pull request closed without merging**